### PR TITLE
Remove traces downloader

### PR DESCRIPTION
**Pull request by janezpodhostnik**
ref: https://github.com/onflow/flow-evm-gateway/issues/638

## Description

Traces will get generated locally.

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-nft/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels 

## Release Notes

- **New Features**
	- No new features added.

- **Bug Fixes**
	- Streamlined error handling for gas price parsing.

- **Refactor**
	- Removed trace-related functionality across various components, simplifying the application and reducing complexity.
	- Updated configuration options by removing trace-related fields.

- **Chores**
	- Adjusted dependency management to categorize certain dependencies as indirect.

- **Tests**
	- Removed unit tests related to trace ingestion functionality.

### CODE REVIEW

**Review by m-Peter — APPROVED** [review]
LGTM! :clap: 
We can also remove the relevant CLI flags for traces on https://github.com/onflow/flow-evm-gateway/blob/main/README.md#configuration-flags.

**Comment by ramtinms** [comment]
I think we should only do this, when we have a local solution, so maybe the PR should target the feature branch.

**Comment by m-Peter** [comment]
> I think we should only do this, when we have a local solution, so maybe the PR should target the feature branch.

@ramtinms Good point, I updated the base branch, to the feature branch we use.

File: README.md [hunk 1]
```diff
@@ -222,14 +222,10 @@ The application can be configured using the following flags at runtime:
 | `stream-limit`                 | `10`                          | Rate-limit for client events sent per second                                             |
 | `rate-limit`                   | `50`                          | Requests per second limit for clients over any protocol (ws/http)                        |
 | `address-header`               | `""`                          | Header for client IP when server is behind a proxy                                       |
-| `heartbeat-interval`           | `100`                         | Interval for AN event subscription heartbeats                                            |
```
Comment (janezpodhostnik) [inline README.md:225]: I forgot to remove this earlier

**Comment by janezpodhostnik** [comment]
@m-Peter Thanks for changing the base, this should definitely only go into the feature branch.

We can hold off on merging this if it causes you any conflicts.

**Review by m-Peter — APPROVED** [review]
LGTM!

**Comment by m-Peter** [comment]
> @m-Peter Thanks for changing the base, this should definitely only go into the feature branch.
> 
> We can hold off on merging this if it causes you any conflicts.

@janezpodhostnik I would say let's merge this, since I also need the changes for enabled the `DebugAPI` :+1:
It doesn't cause me any conflicts.

**Pull request merged**